config/job_conf.xml

```diff
@@ -306,6 +314,7 @@
     <!-- ICL -->
     <tool id="batman-nmr" destination="batman-container"/>
     <tool id="power-analysis" destination="papy-container"/>
+    <tool id="bruker2batman" destination="dynamic-k8s-medium" resources="all"/>
```

Comment:
Good! But for this to work you need to add the adequate lines to the file config/phenomenal_tools2container.yaml (as explained the the wiki entry for setting the job_conf file, where you read about dynamic resources).

Reply:
Basically the same I did here for nrmml2batman: https://github.com/phnmnl/container-galaxy-k8s-runtime/pull/121/commits/df7c3b3832140d45f052595da93f20b468eae080